reference/sectionC_interpretation.inc

```diff
@@ -29,7 +35,7 @@ Units reference
 
 #. The units referencing rules are:
 
-   #. Where there is a :code:`units` element with a :code:`name` attribute identical to the units reference, then the units reference SHALL refer to that :code:`units` element.
+   #. Where, within the same infoset, a :code:`units` element has a :code:`name` attribute identical to the units reference, the units reference SHALL refer to that :code:`units` element.
```

Comment:
So perhaps to replace the old "no rule held to be followed" bit we should just add a line at the end saying "no other values of the name attribute are valid"?

Reply:
Although I see we don't have anything like that for the component refs.

I guess it would be ok to leave this out if we go by the rule that the spec says everything that is valid (stuff it doesn't say is invalid per definition)

Reply:
I'll delete it.